setup.py

```diff
@@ -42,9 +42,9 @@
         "Intended Audience :: Developers",
         "Natural Language :: English",
         "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.9",
+        "Programming Language :: py",
+        "Programming Language :: py :: 3",
+        "Programming Language :: py :: 3.9",
```

Comment:
Was this change intentional? Part of me suspects it might have been an unintentional effect of a "Find all and replace" command...